api/apistruct/struct.go

```diff
@@ -175,12 +181,18 @@ type StorageMinerStruct struct {
 		SectorsRefs   func(context.Context) (map[string][]api.SealedRef, error)       `perm:"read"`
 		SectorsUpdate func(context.Context, abi.SectorNumber, api.SectorState) error  `perm:"write"`
 
-		/*		WorkerStats func(context.Context) (sectorbuilder.WorkerStats, error) `perm:"read"`
+		WorkerConnect func(context.Context, string) error                       `perm:"admin"` // TODO: worker perm
```

Comment:
//TODO

Reply:
Will leave for when we actually rework api permissions